cmd/vela-server/main.go

```diff
@@ -75,6 +75,11 @@ func main() {
 			Usage:   "web ui oauth callback path",
 			Value:   "/account/authenticate",
 		},
+		&cli.StringFlag{
```

Comment:
```suggestion
		&cli.StringSliceFlag{
```

Right? Now I'm curious if the CLI library just parsed it out anyway when you referenced it as `c.StringSlice(` later?